CHANGELOG.md

```diff
@@ -4,6 +4,45 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 1.7.2 (2020-07-07)
+
+This maintenance release contains bugfixes since the 1.7.1 release. We deem it medium
+priority for upgrading.
+
+**Features**
+* #1877 Add support for fast pruning of inlined functions
+
+**Bugfixes**
+* #1908 Fix drop_chunks with unique constraints when cascade_to_materializations is false
+* #1915 Check for database in extension_current_state
+* #1918 Unify chunk index creation
+* #1932 Change compression locking order
+* #1938 Fix gapfill locf treat_null_as_missing
+* #1982 Check for disabled telemetry earlier
+* #1984 Fix compression bit array left shift count
+* #1997 Add checks for read-only transactions
+* #2002 Reset restoring gucs rather than explicitly setting 'off'
+* #2028 Fix locking in drop_chunks
+* #2031 Enable compression for tables with compound foreign key
+* #2039 Fix segfault in create_trigger_handler
+* #2043 Fix segfault in cagg_update_view_definition
+* #2046 Use index tablespace during chunk creation
+* #2047 Better handling of chunk insert state destruction
+* #2049 Fix handling of PlaceHolderVar in DecompressChunk
+
+**Thanks**
+* @akamensky for reporting an issue with telemetry and an issue with drop_chunks
+* @darko408 for reporting an issue with decompression
+* @dmitri191 for reporting an issue with failing background workers
+* @eduardotsj for reporting an issue with indexes not inheriting tablespace settings
+* @fvannee for optimizing pruning of inlined functions
```

Comment:
Would love if we call out community contributions more explicitly as standard practice.

e.g.:

@fvannee for contributing optimizations for pruning inlined functions

(And as @k-rus suggested, a like thanks for Oleg:

@Olernov for contributing a fix for compressing tables with compound foreign keys